test/gmail-adapter.spec.js

```diff
@@ -4,25 +4,34 @@ var Juttle = require('juttle/lib/runtime').Juttle;
 var GmailAdapter = require('../');
 var check_juttle = juttle_test_utils.check_juttle;
 var expect = require('chai').expect;
+var read_config = require('juttle/lib/config/read-config');
```

Comment:
should we make `juttle` a `devDependency` so you don't have to install it yourself when cloning the repo/running tests?